src/Network/Network/Common/NetworkResourceManagerProfile.cs

```diff
@@ -767,6 +767,7 @@ private static void Initialize()
                 cfg.CreateMap<CNM.PSApplicationGatewayRewriteRule, MNM.ApplicationGatewayRewriteRule>();
                 cfg.CreateMap<CNM.PSApplicationGatewayRewriteRuleActionSet, MNM.ApplicationGatewayRewriteRuleActionSet>();
                 cfg.CreateMap<CNM.PSApplicationGatewayHeaderConfiguration, MNM.ApplicationGatewayHeaderConfiguration>();
+                cfg.CreateMap<CNM.PSApplicationGatewayUrlConfiguration, MNM.ApplicationGatewayUrlConfiguration>();
```

Comment:
There is no `MNM.ApplicationGatewayUrlConfiguration` so the build fails. You should commit local SDK package